test-kysect/Task.cs

```diff
@@ -5,6 +5,7 @@ namespace TestKysect
 {
     public class Task
     {
+        private static int _counter;
         public string Name { get; set;  }
         public DateTime Time { get; set;  }
         public int Id { get; set;  }
```

Comment:
```suggestion
        public string Name { get; set; }
        public DateTime Time { get; set; }
        public int Id { get; set; }
```
Чёт лишнее пространство скопилось тут)